hack/benchmark/README.md

```diff
@@ -6,9 +6,39 @@ Benchmark 3-member etcd cluster to get its read and write performance.
 
 1. Start 3-member etcd cluster on 3 machines
 2. Update `$leader` and `$servers` in the script
+3. Apply [connection reuse patch](#connection-reuse-patch-for-boom) to [boom code](https://github.com/rakyll/boom) at rev `79153762` and build boom
```

Comment:
i will specify the git SHA. Current boom is a little diverged from the upstream, but the upstream is making the related change and hopefully we could use upstream soon. I don't want to fork it to add one level of indirection.